enterprise/frontend/src/metabase-enterprise/snippets/components/SnippetCollectionMenu/SnippetCollectionMenu.tsx

```diff
@@ -54,19 +52,19 @@ export function SnippetCollectionMenu({
   };
 
   return (
-    <Box onClick={(e) => e.stopPropagation()} className={S.root}>
+    <Box onClick={(e) => e.stopPropagation()}>
       <Menu position="bottom-end">
         <Menu.Target>
           <Button
-            variant="subtle"
-            p={0}
-            h={20}
-            w={20}
+            w={24}
+            h={24}
             c="text-medium"
-            aria-label={t`Collection options`}
-          >
-            <Icon name="ellipsis" size={16} />
-          </Button>
+            size="compact-xs"
+            variant="subtle"
+            leftSection={<FixedSizeIcon name="ellipsis" size={16} />}
+            aria-label={t`Snippet collection options`}
+            onClick={(e) => e.stopPropagation()}
```

Comment:
We probably don't need this `onClick={(e) => e.stopPropagation()}` since the root Box has it